ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -417,7 +416,7 @@ def node_id(self) -> UUID:
     @cached_property
     def label(self) -> str:
         """Can be overridden as a class attribute to specify a custom label."""
-        return pascal_to_title_case(self._node.__name__)
+        return self._node.__name__
```

Comment:
DevinAI Ok I change my mind. Let's keep this.

Instead, change the codegen logic such that if the node label equals its `pascal_to_title_case` version, we skip code generation